atlasdb-client/src/main/java/com/palantir/atlasdb/table/description/NameComponentDescription.java

```diff
@@ -55,9 +55,16 @@ public NameComponentDescription(String componentName, ValueType type, boolean re
     }
 
     public NameComponentDescription(String componentName,
-                                    ValueType type,
-                                    ValueByteOrder order) {
-        this(componentName, type, order, new UniformRowNamePartitioner(type), null);
+            ValueType type,
+            ValueByteOrder order) {
+        this(componentName, type, order, false);
+    }
+
+    public NameComponentDescription(String componentName,
```

Comment:
Yeah. Let's do this in #2213. Not sure if other products might initialise these directly, since for the most part these are accessed through `TableDefinition`.